src/themes/teams-high-contrast/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,9 @@
+export interface TreeTitleVariables {
```

Comment:
these interface definitions shouldn't be duplicated for `dark` and `high-contrast` themes - please, consider the approach taken for `buttonVariables`, where interfaces for these theme variants are based on the default one: https://github.com/stardust-ui/react/blob/master/src/themes/teams-dark/components/Button/buttonVariables.ts#L4